test/Mvc6Framework45.FunctionalTests/Models/AccountViewModels.cs

```diff
@@ -4,6 +4,7 @@
 using System.Threading.Tasks;
 using System.ComponentModel.DataAnnotations;
 using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.Rendering;
```

Comment:
@Eilon I had the line there, and did "dotnet restore" and "dotnet build" and seems like the build passed with no errors.